packages/shared/lib/contexts/governance/utils/isVotingForProposal.ts

```diff
@@ -1,10 +1,8 @@
-import { get } from 'svelte/store'
 import type { TrackedParticipationOverview } from '@iota/sdk/out/types'
 import { getParticipationsForProposal } from './getParticipationsForProposal'
-import { selectedWalletId } from '@core/wallet'
 
-export function isVotingForProposal(proposalId: string, walletId = get(selectedWalletId)): boolean {
```

Comment:
lets not remove the walletId argument, lets keep these functions generic to any wallet